docs/Common/Enums/index.md

```diff
@@ -230,6 +230,17 @@
 |CID1SET|2||
 |CID2SET|3||
 
+### DeactivateReason
+
+|Name|Value|Description|
+|:---|:----|:----------|
+|AUDIO|0||
+|PHONECALL|1||
+|NAVIGATIONMAP|2||
+|PHONEMENU|3||
+|SYNCSETTINGS|4||
+|GENERAL|5||
```

Comment:
Doesn't seem relevant to the feature.

Reply:
@jacobkeeler, please clarify, should I remove it?

Reply:
Yes

Reply:
Still present

Reply:
@jacobkeeler fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/103/commits/c1863e0adc0ed87d956975819c142c796914f281